xmodule/xml_block.py

```diff
@@ -113,16 +113,15 @@ class XmlMixin:
     xml_attributes = Dict(help="Map of unhandled xml attributes, used only for storage between import and export",
                           default={}, scope=Scope.settings)
 
-    # The attributes will be removed from the definition xml passed
-    # to definition_from_xml, and from the xml returned by definition_to_xml
-
-    # Note -- url_name isn't in this list because it's handled specially on
-    # import and export.
-
```

Comment:
This comment was referring to [metadata_attributes](https://github.com/openedx/edx-platform/blame/ca177570fa6664fef0756b145115110d5e8cf2b3/common/lib/xmodule/xmodule/xml_module.py#L42) and no longer needed.